tests/tracking/test_client.py

```diff
@@ -3502,3 +3503,35 @@ def test_log_spans_and_get_trace_with_sqlalchemy_store(tmp_path: Path) -> None:
         assert child_span.end_time_ns == 1_800_000_000
         assert child_span.attributes.get("operation.type") == "database_query"
         assert child_span.attributes.get("custom.attribute") == "child-value"
+
+
+def test_mlflow_get_trace_with_sqlalchemy_store(tmp_path: Path) -> None:
+    tracking_uri = f"sqlite:///{tmp_path}/test.db"
+
+    with _use_tracking_uri(tracking_uri):
+        client = MlflowClient()
+
+        assert isinstance(client._tracking_client.store, SqlAlchemyTrackingStore)
+
+        with mlflow.start_span():
+            pass
+
+        trace_id = mlflow.get_last_active_trace_id()
```

Comment:
```suggestion
        with mlflow.start_span() as s:
            pass
```

let's remove get_last_active_trace_id